introspection-engine/introspection-engine-tests/Cargo.toml

```diff
@@ -7,6 +7,8 @@ edition = "2018"
 [dependencies]
 sql-datamodel-connector = { path = "../../libs/datamodel/connectors/sql-datamodel-connector" }
 sql-introspection-connector = { path = "../connectors/sql-introspection-connector" }
+migration-connector = { path = "../../migration-engine/connectors/migration-connector" }
+sql-migration-connector = { path = "../../migration-engine/connectors/sql-migration-connector" }
```

Comment:
I wish we had a way to do resets without pulling in the whole sql migration connector, but we currently don't. Something good to keep in mind for later.

Reply:
Yeah, luckily it's only the tests...